WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -332,11 +365,20 @@ private async Task<StatusResponse> FetchStatusAsync()
 	{
 		StatusResponse status = new();
 
+		// Select indexer that's behind the most.
```

Comment:
@yahiheb could you review these changes?

Reply:
LGTM